app/assets/stylesheets/components/_nav.scss

```diff
@@ -7,7 +7,13 @@ nav {
   .nav_left {
     flex: 2;
     h3 {
-      color: $purple;
+      a {
+        color: $purple;
+        text-decoration: none;
+        &:hover {
```

Comment:
What would you think about adding a `&:focus` as well so that the same styles will apply to hover and focus alike?